clang-tools-extra/clangd/ClangdLSPServer.cpp

```diff
@@ -1485,7 +1485,8 @@ ClangdLSPServer::ClangdLSPServer(
     llvm::Optional<OffsetEncoding> ForcedOffsetEncoding,
     const ClangdServer::Options &Opts
 #ifdef INTERACTIVE3C
-    , _3CInterface &Cinter
+    ,
+    _3CInterface &Cinter
```

Comment:
weird

Reply:
(several more below)

Reply:
I agree this is weird, but not bad enough for me to want to manually revert the change after every `clang-format` pass; it might even be considered intended behavior in this edge case.  The case in the `ClangdLSPServer` superclass list where the comma gets moved way to the left is worse and I could file a bug against `clang-format` about that if we want, though I don't know if the maintainers would care too much about its behavior in combination with preprocessor directives like we have here.